src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -1,31 +1,110 @@
 /* eslint-disable no-console */
 import './_dashboard.scss';
-
-import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
-import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
-
+import { Card, CardHeader, CardBody, Divider, PageSection, Title } from '@patternfly/react-core';
 import { Main } from '@red-hat-insights/insights-frontend-components/components/Main';
 import React from 'react';
-import asyncComponent from '../../Utilities/skeletonAsyncCard';
-
-const ComplianceCard = asyncComponent(() => import('../../SmartComponents/Compliance/ComplianceCard'));
-const VulnerabilityCard = asyncComponent(() => import('../../SmartComponents/Vulnerability/VulnerabilityCard'));
+// import asyncComponent from '../../Utilities/skeletonAsyncCard';
```

Comment:
Rather than commenting this out, why not just plop em in where they'll be going eventually?

Reply:
Yep great idea!